composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "php": ">=5.6",
-        "magento/module-sales": "^100.1.2",
+        "magento/module-sales": "^101.0.2",
```

Comment:
This means that the module will _no longer be compatible with version 100_. Is this deliberate? If so, why?

Reply:
Yep good point, amended.